notebooks/introduction_to_tensorflow/solutions/3_keras_sequential_api_vertex_preprocessing_layers.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>There is no need to have <code>create_dataset</code> functions if the only thing they do is to create some numpy arrays. These array can be passed directly to the .fit method, making the code simpler. However, as mentioned earlier, one of the main objective of this lab is to learn how to use the tf.data.Dataset with Keras models. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>